.github/workflows/build.yml

```diff
@@ -28,3 +28,10 @@ jobs:
     with:
       version_override: ${{ inputs.version_override }}
       run_unit_tests: ${{ inputs.run_unit_tests }}
+
+  linux_deb:
```

Comment:
Something we should figure out is whether we want to build a deb file for every PR/merge to main or only on release of a new tagged version. I liken this to the docker image where it's a specialized packaging of the build and we shouldn't need it for testing, outside of specialized cases. With docker I made the call to only do it on release, but I can see an argument for either way.

The advantage of doing it only on release is you could call it from the release yaml which already calculates the official release version from the tag name and passes that through to the current build scripts. As is the build scripts aren't a great source of truth for the version as their either grabbing the AppScaffolding version which is not ment to be end user facing or getting the version from the release workflow.

Reply:
Makes sense, but the version number I need for build-deb is the one that was used to create the tar filename. I need to know which artifact to use with the deb-pkg script, so it has to match.

BTW, where does this value come from? There's no step names 'version' in this job.
https://github.com/rmcrackan/Libation/blob/09118b1ddf634adec35d213bb735604e80773a13/.github/workflows/release.yml#L53

Reply:
In a release context we generate the release version in the release.yml and pass it into the build scripts rather than parsing the value out of the csproj file. If you pass that same version into the new deb packager it would be able to construct the name of the artifact to pull down as long as it's marked as needing the build step first. There's probably something to be said for not defining the naming structure of the artifacts in multiple locations, but I'm not overly concerned about it.

That version is a bug and it should probably be `${{ needs.prerelease.outputs.version }}`. I had to move the version step into it's own prerelease job so that the version could be passed into both the build and docker jobs. I must have missed that one. If you wouldn't mind fixing it since you'll need to test making a release anyway.